Directory.Packages.props

```diff
@@ -16,23 +16,24 @@
     <PackageVersion Include="Microsoft.AspNetCore.Http.Abstractions" Version="[2.1.1,6.0)" />
     <PackageVersion Include="Microsoft.AspNetCore.Http.Features" Version="[2.1.1,6.0)" />
     <PackageVersion Include="Microsoft.CSharp" Version="[4.7.0]" />
-    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="[3.1.0,)" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="[3.1.0,)" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="[3.1.0,)" />
-    <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="[2.1.0,)" />
 
     <!--
-        Typically, the latest stable version of Microsoft.Extensions.Logging should be used here because:
-        1) Each major version bump will have some new API capabilities (e.g. .NET 6 introduced compile-time logging
+        Typically, for the Microsoft.Extensions.* packages relating to DI Abstractions, Hosting Abstractions, and Logging,
+        the latest stable version should be used because:
+        1) Each major version bump will have some new API capabilities (e.g.For Logging, .NET 6 introduced compile-time logging
           source generation, .NET 8 introduced automatic event id generation).
         2) Each minor version bump is normally security hotfixes or critical bug fixes.
         3) Since version 3.1.0, the .NET runtime team is holding a high bar for backward compatibility on
-          Microsoft.Extensions.Logging even during major version bumps, so compatibility is not a concern here.
+          these packages even during major version bumps, so compatibility is not a concern here.
     -->
-    <PackageVersion Include="Microsoft.Extensions.Logging" Version="8.0.0-rc.2.23479.6" />
 
+    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="8.0.0-rc.2.23479.6" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.0-rc.2.23479.6" />
+    <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0-rc.2.23479.6" />
+    <PackageVersion Include="Microsoft.Extensions.Logging" Version="8.0.0-rc.2.23479.6" />
```

Comment:
If we're going to keep all of these in sync could we introduce a new property like `<MicrosoftExtensionsPackageDefaultVersion>8.0.0-rc.2.23479.6</MicrosoftExtensionsPackageDefaultVersion>` they could all use?

Reply:
I would prefer each of them having the version defined explicitly instead of introducing a layer of indirection in determining which version of the package is being used.

Also, in future if only a few of them get patches then we would need lesser changes as we would be able to update the versions of only the affected packages.

Reply:
The indirection might also prevent tools like Dependabot from auto-updating packages.

Reply:
Fair enough! Just FYI we did confirm with @tarekgh that MSE packages can be versioned independently.